src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.ui;
+
```

Comment:
Unrelate commits. You must be familiar with git now to avoid mistakes like this. Practice more with git.